README.md

```diff
@@ -62,6 +64,16 @@ poudriere_config_default:
 When `poudriere_pkg_repo_signing_key` is defined,
 `poudriere_config['PKG_REPO_SIGNING_KEY']` must be set to path to the key file.
 
+## `poudriere_make_conf_files`
+
+This variable is a list of dict. Keys in items are described below.
+
+| Key | Description | Mandatory? |
```

Comment:
s/Mandatory?/Mandatory/ or s/Mandatory/Mandatory?/g for the other tables.

Reply:
fixed in the other branch https://github.com/reallyenglish/ansible-role-poudriere/pull/23#discussion_r149006607